packages/ignition/src/app/structured-data/offer.ts

```diff
@@ -0,0 +1,24 @@
+import { Intangible, Thing } from ".";
```

Comment:
Do not use import like that, it creates redundant references. Next can handle that, so, it works, but it's not ok